vm/src/stdlib/os.rs

```diff
@@ -1542,6 +1542,49 @@ mod posix {
             .map_err(|err| err.into_pyexception(vm))
     }
 
+    #[pyfunction]
+    fn execve(
+        path: PyPathLike,
+        args: Either<PyListRef, PyTupleRef>,
+        env: PyDictRef,
+        vm: &VirtualMachine,
+    ) -> PyResult<()> {
+        let path = ffi::CString::new(path.into_bytes())
+            .map_err(|_| vm.new_value_error("embedded null character".to_owned()))?;
+
+        let args: Vec<ffi::CString> = match args {
+            Either::A(list) => vm.extract_elements(list.as_object())?,
+            Either::B(tuple) => vm.extract_elements(tuple.as_object())?,
+        };
+        let args: Vec<&ffi::CStr> = args.iter().map(|entry| entry.as_c_str()).collect();
+
+        let first = args
+            .first()
+            .ok_or_else(|| vm.new_value_error("execv() arg 2 must not be empty".to_owned()))?;
+
+        if first.to_bytes().is_empty() {
+            return Err(vm.new_value_error("execv() arg2 first element cannot be empty".to_owned()));
```

Comment:
formatting is not uniform
```suggestion
            return Err(vm.new_value_error("execv() arg 2 first element cannot be empty".to_owned()));
```

Reply:
resolved https://github.com/RustPython/RustPython/pull/2101/commits/f1a7014183786035dd695d4248def2f433eda8ea